functional/util.py

```diff
@@ -91,6 +103,77 @@ def is_iterable(val):
     return isinstance(val, collections.Iterable)
 
 
+def split_every(parts, iterable):
+    """
+    Split an iterable in n parts
+
+    >>> l = [1, 2, 3, 4]
+    >>> split_every(l, 2)
+    [[1, 2], [3, 4]]
+
+    :param iterable: iterable to split
+    :param parts: number of chunks
+    :return: return the iterable split in parts
+    """
+    return takewhile(bool, (list(islice(iterable, parts)) for _ in count(0)))
+
+
+def unpack(packed):
+    func, args = serializer.loads(packed)
+    result = func(*args)
+    if isinstance(result, collections.Iterable):
+        return list(result)
+
+
+def pack(func, args):
+    return serializer.dumps((func, args), PROTOCOL)
+
+
+def is_serializable(func, raise_errors=True):
+    if raise_errors is None:
+        raise_errors = True
+    try:
+        serializer.dumps(func, PROTOCOL)
+        return True
+    except (AttributeError, serializer.PicklingError):
+        if raise_errors:
+            raise serializer.PicklingError(
+                "Function {} is not serializable. "
+                "Try installing dill or passing raise_errors to False "
+                "for non-parallel execution when serialization fails."
+                .format(str(func)))
+        return False
+
+
+def parallelize(func, result, processes=None, raise_errors=True):
+    if not is_serializable(func, raise_errors=raise_errors):
+        return func(result)
```

Comment:
Also add a unit test that hits this statement by making it not serializable but `raise_errors=False`